frontend/www/js/omegaup/arena/arena.ts

```diff
@@ -1424,10 +1472,15 @@ export class Arena {
         clarifications.push(clarification);
       }
     } else {
-      r = $('.clarifications tbody.clarification-list tr.template')
-        .clone()
-        .removeClass('template')
-        .addClass('inserted');
+      r = <HTMLElement>(
```

Comment:
```suggestion
      r = <HTMLElement|undefined>(
```

dado que hay un [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), esto debería legalmente poder regresar `undefined`.